sklearn/metrics/_classification.py

```diff
@@ -728,10 +755,13 @@ def jaccard_score(
             Calculate metrics for each instance, and find their average (only
             meaningful for multilabel classification).
 
+        Note that if zero_division is np.nan, such values will be excluded
+        from the average.
```

Comment:
I think we move this into `zero_division`. Currently, when reading the `zero_division` description, the reader needs to scroll up to see what `zero_division=np.nan` does.